<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MianView</class>
 <widget class="QWidget" name="MianView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>航班号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>起点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>终点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>起飞时刻</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>到达时刻</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>剩余座位数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>票价</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>折扣票数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>折扣率</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>航班所属航空公司</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pB_Inquire">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pB_shuaxin">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pB_add">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>增加</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pB_del">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pB_modify">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pB_exit">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_Fightid"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>查询输入</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
